### PR TITLE
Code review for the toDo-list app

### PR DESCRIPTION
**Pull request by Zuhaib042**
**This feature branch consists of toDo-list app code to get reviewed considering the below-mentioned points:-**
- Has the DRY, KISS, and YAGNI principles been followed in the code?
- Has the JS best practices been followed in the code?
- have the HTML and CSS best practices been followed in the code>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import './style.css';
+import TaskStatus from './modules/taskStatus.js';
+import tasks, { inputAdd, listContainer } from './modules/list.js';
+
+window.onload = tasks.displayList();
```
Comment (kaizipaul) [inline index.js]: You can use an Event Listener to render the list
```suggestion
document.addEventListener('DOMContentLoaded', () => {
  tasks.displayList();
});
```

**Review by kaizipaul — CHANGES_REQUESTED** [review]
## Hi Zuhaib! ♻️ Changes required
You've done a great job with your project and you have passed all the best practices and even the DRY, KISS and YAGNI principles
There's just one area I'd suggest to make changes to:

File: index.js (hunk 1 above)
Comment (Zuhaib042) [reply index.js]: thank you so much for your review

**Review by kaizipaul — APPROVED** [review]
Hi! Your changes have been approved! ✅ 
You can now merge your changes to the main branch!
image

**Pull request merged**